docs/quick_start.md

```diff
@@ -115,28 +121,62 @@ In addition to several [standard Web APIs](./api/index.md#web-platform-apis), `j
   Smart functions can invoke other smart functions using `fetch`, similiar to network requests in JavaScript.
   Additionally, new smart functions can be deployed by a smart function using the [`SmartFunction`](./api/smart_function.md) API.
 
-## 2) Deploying your Smart Function
+## 2. Deploying your Smart Function
 
 First we must compile our TypeScript code to JavaScript using:
 
 ```sh
+npm install
```

Comment:
this should be part of the installation guide

Reply:
Disagree, see above